pwiz_tools/Skyline/TestRunner/Program.cs

```diff
@@ -51,6 +51,7 @@
 //         problem.
 //using pwiz.SkylineTestUtil;
 using TestRunnerLib;
+using static System.Windows.Forms.VisualStyles.VisualStyleElement.Rebar;

```

Comment:
I wouldn't expect TestRunner to have any Windows.Forms classes in it. Why this exactly?

Reply:
ugh, just cruft from ReSharper being overeager to help in the midst of me typing something